### PR TITLE
[TACHYON-1428] Documented parts of the tachyon.shell.command package in preparation for the checkstyle rule

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1428

### CODE REVIEW

File: shell/src/main/java/tachyon/shell/command/RmCommand.java [hunk 1]
```diff
@@ -29,6 +29,12 @@
  */
 public final class RmCommand extends WithWildCardPathCommand {
 
+  /**
+   * Constructs a new instance to remove the file specified by argv.
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/RmCommand.java]: argv is at a different level of abstraction from this file - we can just say "to remove the specified file"

Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/RmCommand.java]: or just "to remove a file" since it isn't specified in the constructor

**Comment by uronce-cc** [comment]
Thanks for adding these comments. In general, I feel like most descriptions for the constructor are explanations for the class's usage, what about keeping description of usage only in class comment, and in the constructor comments, just comment the necessary parameters?

**Comment by HorizonNet** [comment]
@aaudiber Thanks for the comments. @uronce-cc You're right. I updated the PR to only comment the necessary parameters.

**Comment by uronce-cc** [comment]
Thanks for adding the comments. LGTM

**Pull request merged**